src/Extensions/LDAPMemberExtension.php

```diff
@@ -264,15 +282,22 @@ public function sync()
     }
 
     /**
-     * Triggered by {@link Member::logIn()} when successfully logged in,
-     * this will update the Member record from AD data.
+     * Triggered by {@link IdentityStore::logIn()}. When successfully logged in,
+     * this will update the Member record from LDAP data.
      */
-    public function memberLoggedIn()
+    public function afterMemberLoggedIn()
     {
         if ($this->owner->GUID) {
-            Injector::inst()
-                ->get(LDAPService::class)
-                ->updateMemberFromLDAP($this->owner);
+            try {
+                Injector::inst()->get(LDAPService::class)->updateMemberFromLDAP($this->owner);
+            } catch (Exception $e) {
+                // If the failure is acceptable, then ignore it and return. Otherwise, re-throw the exception
+                if ($this->owner->config()->allow_update_failure_during_login) {
```

Comment:
Minor (stylistic thing): `$this->owner->config()->get('allow_update_failure_during_login')`

Reply:
Have updated this now.